tasks/connectors/edgescan/lib/edgescan_vulnerability.rb

```diff
@@ -124,6 +124,10 @@ def url?
           !!(location =~ URI::DEFAULT_PARSER.make_regexp)
         end
 
+        def not_ip_range?(specifier)
```

Comment:
Why do we need to check if an IP address is in an IP address range?

Reply:
We're checking that a vulnerability's location specifier is not an ip address range so that we don't use it in the vulnerability's external_id. See comment above.